### PR TITLE
(#23480) Fixed problem with wrong directory path

### LINKED ISSUE #23480: Wrong directory when creating script and file is selected instead folder
**Godot version:**
3.1 alpha 2

**OS/device including version:**
Linux Mint 19

**Issue description:**
When I try to click RMB at folder and click at the new script, then directory is set good, but when I do the same with file instead folder then "/" dissapear

**Steps to reproduce:**
https://streamable.com/uhhxu

### PR DESCRIPTION
**Pull request by Mintormo**

### CODE REVIEW

File: editor/filesystem_dock.cpp [hunk 1]
```diff
@@ -1602,6 +1602,8 @@ void FileSystemDock::_file_option(int p_option, const Vector<String> p_selected)
 			String fpath = path;
 			if (!fpath.ends_with("/")) {
 				fpath = fpath.get_base_dir();
+				if (!fpath.ends_with("/"))
+					fpath = fpath + "/";
 			}
 			make_script_dialog_text->config("Node", fpath + "new_script.gd", false);
```
Comment (akien-mga) [inline editor/filesystem_dock.cpp]: The proper fix would be to use `fpath.plus_file("new_script.gd")` instead of `fpath + "new_script.gd"`.

**Comment by akien-mga** [comment]
That should work, but see my comment for a better fix. It would be nice if you could also amend your commit to use a more explicit commit log - the commit log should tell us in words what the commit is doing. When reading the `git log` locally, "Fixed #23480" is not enough information to know what this commit does, as we're forced to go on GitHub and paste the issue number to find out.

See https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#format-your-commit-logs-with-readability-in-mind and http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html for amending the commit.

File: editor/filesystem_dock.cpp (hunk 1 above)
Comment (Mintormo) [reply editor/filesystem_dock.cpp]: Good. I will try to fix it. Thanks for idea.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**